pkg/kubecost/summaryallocation_json.go

```diff
@@ -0,0 +1,116 @@
+package kubecost
+
+import (
+	"math"
+	"time"
+)
+
+// SummaryAllocationResponse is a sanitized version of SummaryAllocation, which
+// formats fields and protects against issues like mashaling NaNs.
+type SummaryAllocationResponse struct {
```

Comment:
Thoughts on calling this ...Response vs. ...JSON (like the existing `AllocationJSON`)? I don't have a strong opinion.

Reply:
My feeling was just that the type could be used with other protocols, perhaps... I dunno. Today it's only for JSON. I don't feel strongly about it either.

Reply:
Gotcha. Fine as-is, IMO.

Reply:
I think response is a better name - it communicates the intent better and leaves us open for alternative encodings/structures for the future.

Not a big deal either way.